### PR TITLE
docs: reorganize (part 1)

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
The current documentation is difficult to understand where it is written.

- add Home content
- split pages
- categorize the pages
- fix typos

screenshot 2023-09-17 17 54 02

**Online demo:** https://datamweb.github.io/shield/install/

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [] Conforms to style guide

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,23 +1,29 @@
...
-* [Protecting an API with Access Tokens](guides/api_tokens.md)
-* [Mobile Authentication with Access Tokens](guides/mobile_apps.md)
-* [How to Strengthen the Password](guides/strengthen_password.md)
-
-## Addons
-
-* [JWT Authentication](addons/jwt.md)
+## What is Shield?
+
+Shield is an authentication and authorization framework for CodeIgniter 4. While
```
Comment (datamweb) [inline docs/index.md:5]:
```suggestion
Shield is an official authentication and authorization framework for CodeIgniter 4. While
```

**Review by datamweb — APPROVED** [review]
LGTM!

File: docs/index.md (hunk 1 above)
Comment (kenjis) [reply docs/index.md]: Done. 0e4f812558d45fffcf5c0f34d112d8d6381c4c73

**Review by datamweb — APPROVED** [review]
👍

**Comment by kenjis** [comment]
Rebased to resolve conflicts.

**Pull request merged**